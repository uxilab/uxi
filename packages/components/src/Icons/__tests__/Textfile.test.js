import React from 'react';
import Textfile from '../Textfile';

describe('Icon : <Textfile />', () => {
  it('match snapshot', () => {
    expect(shallow(<Textfile />)).toMatchSnapshot();
  });
});

