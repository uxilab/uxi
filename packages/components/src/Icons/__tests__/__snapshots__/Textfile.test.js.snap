// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Textfile /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 12.84 15.75"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <polygon
          points="9.93 9.2 9.93 6.55 2.91 6.55 2.91 9.2 4.11 9.2 4.11 7.75 5.82 7.75 5.82 11.64 4.36 11.64 4.36 12.84 8.48 12.84 8.48 11.64 7.02 11.64 7.02 7.75 8.73 7.75 8.73 9.2 9.93 9.2"
        />
        <path
          d="M1.33,0A1.33,1.33,0,0,0,0,1.33V14.42a1.33,1.33,0,0,0,1.33,1.33H11.51a1.33,1.33,0,0,0,1.33-1.33V3.26L9.58,0ZM11.64,14.55H1.2V1.2H8.73V4.11h2.91Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
